src/frontends/pytorch/src/op_table.cpp

```diff
@@ -482,6 +483,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"aten::masked_scatter_", op::inplace_op<op::translate_masked_scatter>},
         {"aten::matmul", op::translate_1to1_match_2_inputs<opset10::MatMul>},
         {"aten::max", op::translate_max},
+        {"aten::mv", op::translate_mv<opset10::mv>},
```

Comment:
```suggestion
        {"aten::mv", op::translate_1to1_match_2_inputs<opset10::MatMul>},
```

and please remove matmul.cpp as there is no need to create special translator. aten::mv is special case matmul in torch, but for openvino matrix to matrix multiplication or matrix to vector are follow to the same rules and covered by the same operation

Reply:
and replace                                                                                                                                                                                   
  // Ensure matrix input is in float32 dtype
    auto matrix = context.mark_node(std::make_shared<v0::Convert>(context.get_input(0), element::f32));

    // Ensure vector input is in float32 dtype
    auto vector = context.mark_node(std::make_shared<v0::Convert>(context.get_input(1), element::f32));                                             with this                                                                                                                                                                                      auto matrix = context.get_input(0);
    auto vector = context.get_input(1);

Reply:
this replacement is handled automatically inside helper function `translate_1to1_match_2_input`, there is no need to create translate_mv function

Reply:
hey mam What am I doing wrong please tell me

Reply:
// Ensure matrix input is in float32 dtype
 auto matrix = context.mark_node(std::make_shared<v0::Convert>(context.get_input(0), element::f32));
// Ensure vector input is in float32 dtype
auto vector = context.mark_node(std::make_shared<v0::Convert>(context.get_input(1), element::f32));   

replace with this                                                                                                                                                                         
auto matrix = context.get_input(0);
auto vector = context.get_input(1);